goverance.md

```diff
@@ -0,0 +1,67 @@
+- Members *MAY* select additional members through a [super-majority] vote amongst members. This *SHOULD* be supported by a majority of WG Members.
+
+### Role of Chairs
+
+* Primary role of chairs is to run operations and processes governing the WG.
+* The Chairs are responsible for providing technical guidance, for bringing up proposals that have been submitted, and for asking for new proposals to be made.
+* A chair will serve in their role for 2 years from when they started in the role, and *MAY* run again indefinitely.
+* To fill a vacancy for a new chair, the group will announce with at least two week’s notice that a vote for chair will occur. Each company will have one vote. Any group member is eligible to run for Chair.
+* There will be three chairs and their membership will be tracked in [`wg.yaml`]().
+
+
+### Role of Technical Leads
+
+Technical Leads will lead larger streams of work that require sustained technical leadership. Any new proposal should have a TL working as an active sponsor of the proposal, guiding the proposal with the engagement of community. The general list of activities for TL are
+  - Establish new subprojects
+  - Decommission existing subprojects
+  - Resolve cross-subproject technical issues and decisions
+
+There will be two to three TLs and their membership will be tracked in [`wg.yaml`](). TLs are elected by one of the chairs nominating a candidate and creating a PR that adds the TL to [`wg.yaml`](). Members are invited to comment and vote according to the regular proposal process described below.
+
+## Creating, discussing and accepting proposals
+
+Each proposal is unique and might deviate slightly from the process below. For example, a smaller project may each step. For example, a small addition may not require establishing completion criteria. In general, we encourage this process below to be followed to ensure that contributions are in line with the goals of the Working Group.
+### Proposal process
+
+1. **Raise an Issue.** [Create an issue](https://github.com/cn-security/safe/issues/new) in the SAFE repo that outlines the problem to be solved. If possible, include:
+  * The customer impact of the problem
+  * The scope of the work required
+
+1. **Ask the group for collaboration.** Rather than immediately beginning work on a solution, at the next meeting, bring the issue up for discussion, explain it, and ask if others are interested in collaborating. This ensures that solutions are created with multiple perspectives. The outcome of this conversation will be:
+  * The group agrees that this is a problem to solve
+  * There are criteria for completion or failure, and those will be added to the issue.
+  * At least one person is tasked with creating a solution
+  * Those interested in working on the solution either begin work or set up time or expectations with others to begin work.
+  * A TL is agrees to actively sponsor the work.
```

Comment:
I would suggest expanding the TL abbreviation here for clarity; "A Technical Lead is agreed upon to actively sponsor the work."

Do we need a process defined for how the TL is determined?

Reply:
Expanded the abbreviation. I think we defined how the TL is chosen (on line 39); does that feel complete?

Reply:
This is now:
> A Technical Lead agrees to actively sponsor the work.